File: sdks/php/src/Types/PlanResponseBillingInterval.php
<?php

namespace NovaBilling\Types;

enum PlanResponseBillingInterval: string
{
    case Hourly = "HOURLY";
    case Daily = "DAILY";
    case Weekly = "WEEKLY";
    case Monthly = "MONTHLY";
    case Quarterly = "QUARTERLY";
    case Yearly = "YEARLY";
}

File: sdks/ruby/lib/novabilling/types/plan_response_billing_interval.rb
# frozen_string_literal: true

module Novabilling
  module Types
    module PlanResponseBillingInterval
      extend Novabilling::Internal::Types::Enum

      HOURLY = "HOURLY"
      DAILY = "DAILY"
      WEEKLY = "WEEKLY"
      MONTHLY = "MONTHLY"
      QUARTERLY = "QUARTERLY"
      YEARLY = "YEARLY"
    end
  end
end
